### PR TITLE
Add release notes for 1.11.4

### PR DESCRIPTION
**Pull request by adglkh**

### CODE REVIEW

File: announcements/1.11.4.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+None
+
+#### Bugs
+
+A bug in Anbox causes the random webview based application crashes when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgrade the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability and are planning to provide the fix in the Anbox Cloud 1.12 release.
```
Comment (ru-fu) [inline announcements/1.11.4.md:13]:
```suggestion
A bug in Anbox caused random webview-based applications to crash when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgraded the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability, and we are planning to provide a fix in the Anbox Cloud 1.12 release.
```

Comment (ru-fu) [reply announcements/1.11.4.md:13]: Just to confirm: A bug in **Anbox** or a bug in **Anbox Cloud**?

File: announcements/1.11.4.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.11.4.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+None
```
Comment (ru-fu) [inline announcements/1.11.4.md:9]:
```suggestion
None.
```

File: announcements/1.11.4.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+#### Bugs
+
+A bug in Anbox causes the random webview based application crashes when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgrade the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability and are planning to provide the fix in the Anbox Cloud 1.12 release.
+
+
+#### Upgrade instructions
+
+This release does not include any charm changes. Only images are updated and will be automatically synchronized to your Anbox Cloud deployment via the Canonical image server. See (Image Management)[https://anbox-cloud.io/docs/howto/manage/images]  for more details.
```
Comment (ru-fu) [inline announcements/1.11.4.md:18]:
```suggestion
This release does not include any charm changes. Only the images are updated, and they are automatically synchronised to your Anbox Cloud deployment via the Canonical image server. See [Manage images](https://anbox-cloud.io/docs/howto/manage/images) for details.
```

File: release-notes.md [hunk 4]
```diff
@@ -2,6 +2,20 @@
...
+
+## 1.11.4 (November 1 2021)
+
+### New features & improvements
+
+None
+
+### Bugs
+
+A bug in Anbox causes the random webview based application crashes when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgrade the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability and are planning to provide the fix in the Anbox Cloud 1.12 release.
```
Comment (ru-fu) [inline release-notes.md:15]:
```suggestion
None.

### Bugs

A bug in Anbox caused random webview-based applications to crash when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgraded the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability, and we are planning to provide a fix in the Anbox Cloud 1.12 release.
```

Comment (ru-fu) [reply release-notes.md:15]: (same changes as for the announcement)

**Review by ru-fu — CHANGES_REQUESTED** [review]

File: announcements/1.11.4.md (hunk 1 above)
Comment (adglkh) [reply announcements/1.11.4.md:13]: hmm, I think it's better to call out *Anbox* here than *Anbox Cloud* as it's caused by the lower stack. Maybe @morphis has a better idea.

File: release-notes.md (hunk 4 above)
Comment (adglkh) [reply release-notes.md:15]: +1

**Review by ru-fu — APPROVED** [review]

File: announcements/1.11.4.md (hunk 1 above)
Comment (morphis) [reply announcements/1.11.4.md]: Lets call it the `Anbox runtime` here. Anbox Cloud is the entire stack whereas Anbox is one particular thing in the stack. Calling it `Anbox runtime` should make the distinction clear.

Comment (adglkh) [reply announcements/1.11.4.md]: +1

File: announcements/1.11.4.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+None.
+
+#### Bugs
+
+A bug in Anbox runtime caused random webview-based applications to crash when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgraded the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability, and we are planning to provide a fix in the Anbox Cloud 1.12 release.
```
Comment (ru-fu) [inline announcements/1.11.4.md:13]:
```suggestion
A bug in the Anbox runtime caused random webview-based applications to crash when upgrading the webview.apk to [94.0.4606.80](https://chromereleases.googleblog.com/2021/10/chrome-for-android-update.html). For this reason, we downgraded the webview version to [93.0.4577.82](https://chromereleases.googleblog.com/2021/09/chrome-for-android-update.html) for stability, and we are planning to provide a fix in the Anbox Cloud 1.12 release.
```
(In both files)

**Review by ru-fu — APPROVED** [review]

**Review by morphis — APPROVED** [review]
LGTM

**Pull request merged**